frontend/server/src/DAO/Contests.php

```diff
@@ -132,7 +136,7 @@ final public static function getByTitle(string $title) {
     final public static function getByAliasWithExtraInformation(string $alias): ?array {
         $sql = '
                 SELECT
-                    c.*,
+                    join(', ', array_keys(\OmegaUp\DAO\VO\Contests::FIELD_NAMES)),
```

Comment:
El problema con esta parte es que `join(....)` es código de PHP y la variable `$sql` es una cadena. Entonces necesitamos concatenar lo que regresa el `join()`. Adicionalmente le concatene la variable `c` para no afectar el resto del código.

```suggestion
                    'c.' . join(', c.', array_keys(\OmegaUp\DAO\VO\Contests::FIELD_NAMES)) .  ' ' . ',' .
```

Reply:
Para que sea un poco más legible el código, pudieras utilizar una variable extra que se encargue de armar la cadena con los campos, para que después sólo sea necesario concatenarla. Por ejemplo:
```php
        $fields = join(
            '',
            array_map(
                fn (string $field): string => "c.{$field}, ",
                array_keys(
                    \OmegaUp\DAO\VO\Contests::FIELD_NAMES
                )
            )
        );
        $sql = '
                SELECT
                    ' . $fields . '
                    p.scoreboard_url,
                    p.scoreboard_url_admin
                FROM
                    Contests c
                INNER JOIN
                    Problemsets p
                ON
                    p.problemset_id = c.problemset_id
                WHERE
                    c.alias = ?
                LIMIT 1;';
```